src/global/responses/errors.ts

```diff
@@ -26,6 +26,17 @@ export class BadRequestErrorResponse {
     statusCode: number;
 }
 
+export class BadRequestErrorArrayResponse {
+    @ApiProperty({ isArray: true })
```

Comment:
Should we add an example in the message?
![Screenshot 2024-01-11 at 3 49 05 PM](https://github.com/chingu-x/chingu-dashboard-be/assets/80556079/eccb56fe-1c2d-4a9d-963c-3003fa0ff44b)